vivisect/qt/funcgraph.py

```diff
@@ -314,7 +315,14 @@ def vqSetSaveState(self, state):
     def updateWindowTitle(self):
         ename = self.getEnviNavName()
         expr = str(self.addr_entry.text())
-        self.setWindowTitle('%s: %s' % (ename,expr))
+        try:
+            va = self.vw.parseExpression(expr)
+        except:
+            va = 0
```

Comment:
This is a "using 0 when you mean None" instance here. Probably want to avoid that because there will be instances of embedded systems with memory at addr==0 and it would be *really* confusing for it to be magically printing a symbol when what it really means is "syntax error".  It would maybe even be most helpful to put some small bit of the exception info into the title ( fail fast and obvious ) rather than masking/hiding the error.